src/server/main_service.cc

```diff
@@ -193,7 +193,7 @@ void MultiCleanup(ConnectionContext* cntx) {
     exec_info.preborrowed_interpreter = nullptr;
   }
   UnwatchAllKeys(cntx->ns, &exec_info);
-  exec_info.Clear();
+  exec_info.Clear(cntx->conn());
```

Comment:
you won't need to pass connection once you move stats into ServerState::Stats

Reply:
yes removed now